src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -194,28 +194,28 @@
     <value>Multiple panes</value>
   </data>
   <data name="TabCloseSubMenu" xml:space="preserve">
-    <value>Close...</value>
+    <value>Close</value>
   </data>
   <data name="TabCloseAfter" xml:space="preserve">
-    <value>Close Tabs to the Right</value>
+    <value>Close tabs to the right</value>
   </data>
   <data name="TabCloseOther" xml:space="preserve">
-    <value>Close Other Tabs</value>
+    <value>Close other Tabs</value>

```

Comment:
```suggestion
    <value>Close other tabs</value>
```
Shouldn't "Tabs" be changed to "tabs" too? The open issue said that we should use "Sentence-style capitalization".